src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -580,7 +578,26 @@ const TempRHGraph: React.FunctionComponent<Props> = ({
         textY: legendY + 3,
         radius: 0.5
       })
-
+      d3Utils.addLegend({
+        svg,
+        text: 'High Res Model Temp',
+        color: styles.highResModelTempDotColor,
+        fill: 'none',
+        shapeX: legendX += 127,
```

Comment:
We're going to have to look at using layouts moving forward to achieve the placement of objects. Specifying the pixel positions in this way is cumbersome. I googled a bit, and this seems like a good starting point: https://www.d3indepth.com/layouts/

Reply:
I agree, we have to revisit this at some point for sure.